<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>256</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>256</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Livestreamer - kadse GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Log</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="logTextEdit">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier werden die Meldungen, die Livestreamer ausgibt, eingetragen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="placeholderText">
          <string>Log...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Stream</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="5" column="0">
          <widget class="QComboBox" name="streamComboBox">
           <property name="toolTip">
            <string>URL auswählen oder eintragen und dann Enter drücken.</string>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Qualität</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>URL eingeben/auswählen:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="qualityComboBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Gewünschte Qualität auswählen.</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="saveUrlPushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;URL abspeichern, um sie später in der Liste auswählen zu können, anstatt sie erneut eintragen zu müssen.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>URL speichern</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="streamPushButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stream starten.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Stream</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStreams">
    <property name="title">
     <string>Streams</string>
    </property>
    <addaction name="actionAllStreams"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuData"/>
   <addaction name="menuStreams"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <action name="actionAddStream">
   <property name="text">
    <string>Hinzufügen...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="actionAllStreams">
   <property name="text">
    <string>Alle Streams...</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Speichern</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>streamComboBox</tabstop>
  <tabstop>qualityComboBox</tabstop>
  <tabstop>logTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
